help/C/Help_ch_Transactions.xml

```diff
@@ -899,180 +905,543 @@ Author:
+            <para>Click on <guibutton><emphasis role="bold">Close</emphasis></guibutton> to exit the wizard.</para>
+          </listitem>
+          </itemizedlist>
         </para>
       </sect3>
     </sect2>
+    
     <sect2 id="trans-import-ofx">
       <title>Import OFX/QFX</title>
-      <para>This opens a file selection dialog. Navigate to the file you wish to import, select a file with 
-        the appropriate extension (.ofx  or .qfx), then press the <guibutton>Import</guibutton> button button.</para>
-      <para>Gnucash opens an Account Selection dialog to select an account in your CoA corresponding to data source. Select
-        the appropriate account from the account tree and press the <guibutton>OK</guibutton> button. On subsequent import of files from the 
-        same source (identified by tags in the file), the source is remembered and the account selection dialog is not
-        displayed.</para>
+      <para><guimenu>File</guimenu>-><guisubmenu>Import</guisubmenu>-><guimenuitem>Import OFX/QFX transactions</guimenuitem>
+        opens a file selection dialog. Navigate to the file you wish to import, select a file with 
+        the appropriate extension (.ofx  or .qfx), then press the <guibutton><emphasis role="bold">Import</emphasis></guibutton> button.</para>
+      <para>&app; then opens an Account Selection dialog to select an account in your CoA to match to an account name in the import file. Select
+        the appropriate account from the account tree and press the <guibutton><emphasis role="bold">OK</emphasis></guibutton> button. On subsequent import of files from the 
+        same source account, the mapped &app;account is remembered and the account selection dialog is not displayed.
+      </para>
       <para>The generic import transaction matcher dialog is opened next. See the <link linkend="trans-import-matcher">
-            Import Matcher</link> section (common to both OFX/QFX and CSV import formats) following the Import CSV
-            section to continue the import process.
+        Import Matcher</link> section (common to both OFX/QFX and CSV import formats) following the Import CSV
+        section to continue the import process.
       </para>      
     </sect2>
+    
     <sect2 id="trans-import-csv">
       <title>Import CSV</title>
-      <para>Clicking on <guimenuitem>Import CSV...</guimenuitem> in the Import menu will bring up the Import Assistant dialog. The first step brings up a file selection 
-            dialog. Navigate to the location where the file you wish to import is located and select the file to import then click the 
-            <guibutton>OK</guibutton> button.
+      <para>Clicking on <emphasis role="bold"><guimenu>File</guimenu>-><guisubmenu>Import</guisubmenu>-><guimenuitem>Import Transactions from CSV</guimenuitem></emphasis>
+        will bring up the CSV Transaction Import Assistant dialog illustrated below.
```

Comment:
Mention lowest level of exchange format? All others are structured financial data. CSV is a plain table.

Reply:
Will do

Reply:
Added a bit to the introductory para in Import Formats pointing this out.